multichain-testing/package.json

```
@@ -0,0 +1,92 @@
+{
+  "name": "@agoric/multichain-testing",
+  "version": "0.1.0",
```

We shouldn't be publishing this, so it doesn't need a version. We shouldn't be importing from it so it doesn't need a name.

It does need this to prevent publishing:
```suggestion
  "private": true,
```

---

Was thinking external developers might want to depend on this at some point, but suppose it could be viewed as boilerplate instead. Can cross that bridge when we come to it.